XS✔ ◾ Improved title "start-answer-with-yes-or-no" (#6757)
SSWConsulting/SSW.Rules.Content: rules/start-your-answer-with-yes-or-no-then-say-your-opinion/rule.md

---
type: rule
archivedreason: 

> title: Clarity - Do you start your answer with ‘Yes’ or ‘No’, then give more info?

Reviewer: ```suggestion
title: Do you answer with ‘Yes’ or ‘No’ for clarity before providing extra information?
```
